presentation/src/main/java/com/knocklock/presentation/lockscreen/receiver/StartApplicationReceiver.kt

```diff
@@ -0,0 +1,48 @@
+package com.knocklock.presentation.lockscreen.receiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.util.Log
+import dagger.hilt.android.AndroidEntryPoint
+
+/**
+ * @Created by 김현국 2022/12/13
+ * @Time 4:00 PM
+ */
+
+@AndroidEntryPoint
+class StartApplicationReceiver(
+    private val context: Context,
+    private val openLockScreen: OpenLockScreen
+) : BroadcastReceiver() {
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        startLockScreen(context)
+    }
+
+    private fun startLockScreen(context: Context?) {
+        if (context != null) {
+            try {
+                openLockScreen.openLockScreenByIntent()
+            } catch (e: Exception) {
+                Log.e("Receiver Exception", e.stackTraceToString())
+            }
+        }
+    }
+    fun stop() {
+        context.unregisterReceiver(this)
+    }
+
+    fun start() {
```

Comment:
좀 더 명확히 네이밍하면 좋을 것 같아요!!
```suggestion
    fun unregisterReceiver() {
        context.unregisterReceiver(this)
    }

    fun registerReceiver() {
```

Reply:
Receiver 클래스들의 method를 위와 동일하게 전부 변경했습니당

Reply:
@jaeryo2357  [2b3fe7d](https://github.com/Knock-Lock/Knock-Lock/pull/69/commits/2b3fe7dec11fea11c2c863e30ac3e8f89990cb10)
[6a665e8](https://github.com/Knock-Lock/Knock-Lock/pull/69/commits/6a665e8a3ecce6e49d481a7c75e133be02c63c9d) 에서 반영했습니당